verilog/formatting/tree_unwrapper.cc

```diff
@@ -2060,8 +2060,9 @@ class MacroCallReshaper {
 
     paren_group_ = FindDirectChild(main_node_,
                                    OriginTagIs{NodeTag(NodeEnum::kParenGroup)});
+
+    // Macro can be used without parentheses
     if (!paren_group_) {
-      LOG_PARTITION_BUG("paren_group not found.");
```

Comment:
This message has been first added in https://github.com/chipsalliance/verible/commit/56600a7ed5125a56b9ee9f0c5a5995bfa4366996 so maybe @mglb can comment on the reason for it, and maybe if it is ok in this case.

Reply:
It is OK, this was a wrong assumption.